### DISCUSSION TITLE
Fork per Tienimi Bordone

### DISCUSSION THREAD

**Discussion by paolozanchi** [pdonadeo/e-uscito-joypad#53] [Ideas]
Non ho idea di quanto sia grande la fetta di utenza condivisa tra i due podcast ma io la butto lì... se si facesse un fork di questo progetto (backend e frontend) per il podcast Tienimi Bordone?

Trovo che l'idea di base (classificare gli argomenti di una puntata) sia la stessa, forse anche più facile visto il numero minore di argomenti trattati in ogni puntata (compensati però dal numero enorme di puntate).

L'unica complicazione che vedo è il fatto che gli episodi siano disponibili solo per gli abbonati, quindi forse l'integrazione con il player sarebbe più difficile/impossibile.

**Comment by nsalmoria** [comment]
Un database di Tienimi Bordone sarebbe una cosa meravigliosa.

**Reply by pdonadeo** [reply]
Ciao Nicola, una domanda completamente off topic rispetto a tutto questo: ma su scrivevi, oltre 20 anni fa, in una (meravigliosa) mailing list che si chiamava matenigmici?

**Reply by nsalmoria** [reply]
Oddio, forse (magari anche 30 anni fa...) però ricordo principalmente il forum Matenigmici di MC-link. Proprio un'altra era geologica...

**Reply by pdonadeo** [reply]
È la stessa era geologica allora. Lieto di averti incontrato nuovamente 😄 C'ero anch'io su MC-link e specialmente in Matenigmici, da cui quella mailing list è derivata.

**Reply by nsalmoria** [reply]
Piacere mio, che tuffo in ricordi di un tempo a cui non penso più...

**Reply by DGuidi** [reply]
aspetta aspetta, ma tu sei lo stesso nicola salmoria che ha pure la pagina su wikipedia? dimmi di si, dimmi di si

**Reply by nsalmoria** [reply]
Sì ma penso che stiamo andando troppo per la tangente :-)

**Reply by DGuidi** [reply]
:)
![download](https://user-images.githubusercontent.com/216627/197775183-d41e2bd4-492e-4ded-ab81-ef3338b7f8a6.jpg)

**Comment by pdonadeo** [comment]
Per quanto l'idea sia meravigliosa in se penso che "categorizzare Tienimi Bordone" sia un'impresa impossibile. Forse una ricerca full text su tutti i transcript avrebbe più senso, ma il costo di ottenere la trascrizione (automatica) di oltre 700 episodi mi sembra proibitivo per un progetto hobbistico.

Poi vengono i problemi di copyright: rendere a tutti disponibile (di fatto) la trascrizione di contenuti solo per abbonati mi sembra una scelta legalmente molto azzardata, che io certamente non voglio fare 😃 

Rimane insomma un argomento di cui discutere eventualmente con la redazione de Il Post per integrare questo "motore di ricerca" dentro al Post stesso, solo per abbonati.

Bello, ma la vedo dura.

**Reply by nsalmoria** [reply]
Già anche soltanto una ricerca sui titoli delle puntate sarebbe utile. Però certo, è una cosa di cui si dovrebbe occupare il Post, visto che è un contenuto per i soli abbonati.
